### DISCUSSION TITLE
Background Color On Page Keeps Fluctuating

### DISCUSSION THREAD

**Discussion by samarzayan** [quarto-dev/quarto-cli#2973] [Q&A]
I have set up a different background color on one page but when I shift from a different page to it in preview mode. It sort of resets with the default general background. I was wondering if its just an error or is it getting messed up.

**Comment by mcanouil** [comment]
Hi,

can you share some reproducible example that exhibit your issue? 😉
Without it's going to be hard (or impossible) to really help and figure out what is possibly wrong.

**Reply by samarzayan** [reply]
Hi, thanks for assisting. I figured it out actually. 
The page was setting itself to the theme(s) set in the `_quarto.qmd` file, which applies to all the pages. I created a separate CSS theme in a different file for that page and it stayed intact.
Thanks again.
